src/lib.rs

```diff
@@ -50,6 +51,7 @@ extern crate hyper;
 #[cfg(unix)]
 extern crate nix;
 extern crate alloc;
+extern crate chrono;
```

Comment:
I'm not sure. Though `Time` is deprecated, but it just works fine. And this crate depends on time too.